modules/mediawiki/data/xml_dump.yaml

```diff
@@ -11,6 +11,7 @@ nenawikiwiki: 'weekly'
 nissanecuwiki: 'fortnight'
 nonbinarywiki: 'monthly'
 renaissancewiki: 'monthly'
+schattenvonskelos: 'fortnight'
```

Comment:
I think this should have wiki at the end of the name.

Reply:
its already been reverted @paladox